frontend/src/app/trees/forests/tree-guidelines/sidebar-view.component.html

```diff
@@ -1,78 +1,91 @@
 <section id="sidebar-page" class="usa-section usa-content">
   <div class="usa-grid breadcrumbs">
-    <a routerLink="/xmas-trees/forests">Christmas tree permits</a> > <span><a routerLink="/xmas-trees/forests">Find a forest</a></span><span> > {{ forest.forestName }} National Forest{{ forest.forestName.indexOf('and') > 0 ? 's' : ''}}</span>
+    <a routerLink="/christmas-trees/forests">Christmas tree permits</a> > <span><a routerLink="/christmas-trees/forests">Find a forest</a></span><span> > {{ forest.forestName }} National Forest{{ forest.forestName.indexOf('and') > 0 ? 's' : ''}} Christmas tree permit information</span>
```

Comment:
It really feels time to make the sidebars a less hard coded view and more of a component that leverages iterative generation now that we're using them so frequently.

Reply:
@lauraGgit created new story to address this: https://github.com/nciinc/fs-intake-module/issues/124